build/release-tars/BUILD

```diff
@@ -1,8 +1,43 @@
 package(default_visibility = ["//visibility:public"])
 
+load(
+    "//build:platforms.bzl",
+    "for_platforms",
+    "go_platform_constraint",
+    "CLIENT_PLATFORMS",
+    "NODE_PLATFORMS",
+    "SERVER_PLATFORMS",
+    "TEST_PLATFORMS",
+)
 load("@io_k8s_repo_infra//defs:build.bzl", "release_filegroup")
 load("@io_k8s_repo_infra//defs:pkg.bzl", "pkg_tar")
 
+# Bazel doesn't make the output filename
```

Comment:
this is basically the pattern I have to use all over the place, but I'm not sure where a good spot to put this is. Maybe in the bazel docs on k/community?